harvard-lts/CURIOSity
# added focus outline via CSS

**JIRA Ticket**: [LTSMAINT-679](https://jira.huit.harvard.edu/browse/LTSMAINT-679)

# What does this Pull Request do?
Adds a focus state outline as a user navigates to and through the search field of a CURIOSity exhibit and the array of filters in the 'Limit Your Search' column

# How should this be tested?

* Rebuild container using add_focus_search_filter branch
* Go to any exhibit in your local instance that uses the search and filter features of Spotlight
* Use the Tab key and Shift + Tab keys combination to navigate through the page's elements
* Confirm you can see a blue outline around the search field in the exhibit's main nav bar, the button that initiates the search, and the categories of filters found in the 'Limit Your Search' column along the left side of the page; this indicates to the user where the cursor is located on the screen

# Test coverage
Yes/No: Are changes in this pull request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@phil-plencner-hl @dl-maura
